python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -94,7 +94,11 @@ def __init__(self, parent, schema_generation=0, bg_task_manager=None, include_ro
             supplied arg would look like: ``include_root="Project Publishes"``.
             If ``include_root`` is `None`, no root item will be added.
         """
-        super(ShotgunHierarchyModel, self).__init__(parent, bg_task_manager)
+        super(ShotgunHierarchyModel, self).__init__(
+            parent,
+            bg_load_thumbs=True,
```

Comment:
Does this change the behavior or is this the default? Any concerns here?

Reply:
possibly but this is a good thing. Previously it would pass the bg task manager class instance into a bool and that would be the state of bg_load_thumbnails - so in the cases where a task manager was explicitly passed in, there would be thumbs otherwise not.